src/components/interaction-designer/clipboard/ClipboardRoot.vue

```diff
@@ -69,16 +86,37 @@ export default {
   created() {
     this.initializeFlowRunner()
   },
+  computed: {
+    isFocused(index) {
+      return this.isBlockFocused(index)
+    },
+  },
   methods: {
     ...mapGetters('flow', ['getFlowsState']),
-    ...mapActions('clipboard', ['setSimulatorActive']),
+    ...mapGetters('clipboard', ['getBlocksData', 'isBlockFocused', 'getBlockPrompt']),
```

Comment:
these getters should be under `computed`

Reply:
Done!